pkg/timezone/timezone_windows.go

```diff
@@ -0,0 +1,181 @@
+//go:build windows
+
+package timezone
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+// Timezone returns the local timezone set in registry. If not, an error is returned.
+func Timezone() (string, error) {
+	key, err := registry.OpenKey(
+		registry.LOCAL_MACHINE,
+		`SYSTEM\CurrentControlSet\Control\TimeZoneInformation`,
+		registry.QUERY_VALUE)
+	if err != nil {
+		return "", fmt.Errorf("failed to open registry key")
+	}
+
+	defer key.Close()
+
+	tzwin, _, err := key.GetStringValue("TimeZoneKeyName")
+	if err != nil {
+		return "", fmt.Errorf("failed to get key value from registry")
+	}
+
+	// For some reason this returns a string with loads of NUL bytes at
+	// least on some systems. I don't know if this is a bug somewhere, I
+	// just work around it.
+	tzwin = strings.ReplaceAll(tzwin, "\x00", "")
+
+	if tz, ok := timezones[tzwin]; ok {
+		return tz, nil
+	}
+
+	return "", fmt.Errorf("windows timezone '%s' not found", tzwin)
+}
+
```

Comment:
This map looks the same as [tzlocal/windows_tz.py](https://github.com/regebro/tzlocal/blob/d5fb741969269c1cc0f4c1a87a66c10426863761/tzlocal/windows_tz.py) but that file contains many more zone mappings. Should we add those here?